### PR TITLE
Abi update & auto kick & kick Func

### PR DESCRIPTION
**Pull request by parkkoon**

### CODE REVIEW

**Review by c0smic0wl — APPROVED** [review]
:owl:

File: app/javascripts/components/playSlot/reducer.jsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const PLAY_SLOT_INITIAL_STATE = fromJS({
   hasError: false,
   betSize: Big(20),
   lineNum: 20,
-  deposit: 0,
+  deposit: Big(0),
```
Comment (fish-sushi) [inline app/javascripts/components/playSlot/reducer.jsx:15]: :+1:

File: app/javascripts/components/watchSlot/actions.jsx [hunk 2]
```diff
@@ -91,11 +132,6 @@ export function getSlotMachine(slotAddress, playerAddress) {
       const slotMachineContract = Web3Service.getSlotMachineContract(slotAddress);
       const slotInfo = await Web3Service.getSlotMachineInfo(slotMachineContract);
 
-      // Set occupied state
```
Comment (fish-sushi) [inline app/javascripts/components/watchSlot/actions.jsx:94]: Is there lines aren't needed?

File: app/javascripts/components/watchSlot/reducer.jsx [hunk 3]
```diff
@@ -20,6 +20,7 @@ export const WATCH_SLOT_INITIAL_STATE = fromJS({
   initQueue: [],
   confirmQueue: [],
   chainIndex: 0,
+  player: '',
```
Comment (fish-sushi) [inline app/javascripts/components/watchSlot/reducer.jsx]: Please change this variable to `playerAddress`

File: app/javascripts/helpers/web3Service.jsx [hunk 4]
```diff
@@ -743,6 +658,17 @@ class Web3Service {
       return slotMachineContractAddress;
     });
 
+    setInterval(() => {
```
Comment (fish-sushi) [inline app/javascripts/helpers/web3Service.jsx]: Add removeInterval logic or make it works only on a banker side.

**Review by fish-sushi — CHANGES_REQUESTED** [review]

File: app/javascripts/components/watchSlot/actions.jsx (hunk 2 above)
Comment (parkkoon) [reply app/javascripts/components/watchSlot/actions.jsx:94]: Yes. occupied state will be used for being valid at slot Game.

File: app/javascripts/components/watchSlot/reducer.jsx (hunk 3 above)
Comment (parkkoon) [reply app/javascripts/components/watchSlot/reducer.jsx]: OK

File: app/javascripts/helpers/web3Service.jsx (hunk 4 above)
Comment (parkkoon) [reply app/javascripts/helpers/web3Service.jsx]: OK. I will

**Pull request merged**